homeassistant/components/lacrosse_view/sensor.py

```diff
@@ -52,6 +58,46 @@ class LaCrosseSensorEntityDescription(
     "kilometers_per_hour": "km/h",
     "inches": "in",
 }
+SENSOR_DESCRIPTIONS = {
+    "Temperature": LaCrosseSensorEntityDescription(
+        key="",
+        device_class=SensorDeviceClass.TEMPERATURE,
+        name="Temperature",
+        state_class=SensorStateClass.MEASUREMENT,
+        value_fn=get_value,
+        native_unit_of_measurement="°C",
```

Comment:
Please use our common constants for the units.

https://github.com/home-assistant/core/blob/4a938ec33ee8df17e619fc64a519ef7c2ff856d8/homeassistant/const.py#L508